dockerfiles/README.md

```diff
@@ -72,6 +71,25 @@
   
   docker run -it onnxruntime-ngraph
   ```
+## ONNX Runtime Server (Preview)
+#### Linux 16.04
+
+1. Build the docker image from the directory that contains our server "onnxruntime_server"
+  ```
+  docker build -t {docker_image_name} .
+  ```
+  
+2. Run the ONNXRuntime server in Docker image
+
+  ```
+  docker run -it -v {localModelAbsoluteFolder}:{dockerModelAbsoluteFolder} -e MODEL_ABSOLUTE_PATH={dockerModelAbsolutePath} -p {your_local_port}:8001 {imageName}
+  ```
+3. Send the request to server
+
+ - Send the request to the docker through the binding local port:
```

Comment:
1) You don't have to use "- " at beginning. 
2) You may want to have a link to the full usage doc here. 
```suggestion
 Send the request to the docker through the binding local port. Here is the full [usage document](https://github.com/Microsoft/onnxruntime/blob/master/docs/ONNX_Runtime_Server_Usage.md).
```

Reply:
fixed